recipe/meta.yaml

```
@@ -7,41 +7,55 @@ package:
 source:
   fn: {{ version }}.tar.gz
   url: https://github.com/p1c2u/openapi-spec-validator/archive/{{ version }}.tar.gz
-  sha256: f004bc1133b4c1ba1bc49577bf454be5e1bcc7ea64b21b7a112678da6577d35a
+  sha256: 9d7201cbef1e584f7323c96dc3e45d1a8d1f387a5b0ed808948e094fee328ee2
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - openapi-spec-validator = openapi_spec_validator.__main__:main
 
 requirements:
   host:
     - pip
+    - poetry-core >=1.0.0
     - python
-    - setuptools
-    - pyyaml
 
   run:
+    # keep in sync with the poetry-generated setup.py
+    - jsonschema >=4.18.0,<5.0.0
+    - openapi-schema-validator >=0.6.0,<0.7.0
+    - importlib-resources >=5.8,<7.0  # [py<39]
+    - jsonschema-path >=0.3.1,<0.4.0
+    - lazy-object-proxy >=1.7.1
```

```suggestion
    - lazy-object-proxy >=1.7.1,<2
```
https://inspector.pypi.io/project/openapi-spec-validator/0.7.1/packages/2b/4d/e744fff95aaf3aeafc968d5ba7297c8cda0d1ecb8e3acd21b25adae4d835/openapi_spec_validator-0.7.1-py3-none-any.whl/openapi_spec_validator-0.7.1.dist-info/METADATA#line.27